pom.xml

```diff
@@ -10,21 +10,20 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <sonar.version>8.6.1.40680</sonar.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-plugin-api</artifactId>
-      <version>${sonar.version}</version>
+      <version>7.9</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.sonarsource.sonarqube-plugins.cxx</groupId>
       <artifactId>sonar-cxx-plugin</artifactId>
       <type>sonar-plugin</type>
-      <version>0.9.7-SNAPSHOT</version>
+      <version>2.1.1-SNAPSHOT</version>
```

Comment:
Think this must be the version of the plugin. The "-SNAPSHOT" is replaced with the build number.